### PR TITLE
CONTRIBUTING: add reference to commit guidelines

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

**Review by kursatkobya — APPROVED** [review]
Maybe using "we are following" instead of "We try to" would be better for being more strict for upcoming commits

**Comment by tobsan** [comment]
@kursatkobya yes, I thought about that at first, and we should be strict, but let's be honest: we're gonna screw it up from time to time as well.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ We are happy to accept contributions. To accept a contribution, we require two t
   https://www.clahub.com/agreements/Pelagicore/softwarecontainer
   - We don't like CLAs but we have to have one. We'll remove it as soon as we can.
 
+We try to follow the [https://wiki.gnome.org/Git/CommitMessages](GNOME guidelines) for commit
```
Comment (thenor) [inline CONTRIBUTING.md]: I do agree with Kursat. My suggestion is to just rephrase this to "We aim to follow".

**Review by thenor — CHANGES_REQUESTED** [review]

**Review by thenor — APPROVED** [review]

**Comment by rpannek** [comment]
Why is the "CONTRIBUTING" in this commit all upper case? I read the linked https://wiki.gnome.org/Git/CommitMessages which explicitly says: "You can add a tag in lower cases with a colon before the rest." So we did it wrong in the first commit ;)

**Comment by rpannek** [comment]
And btw in markdown you do `[link text](http://example.com)` not the other way around like in this commit.

**Comment by tobsan** [comment]
@rpannek as I said: we're gonna screw it up from time to time. See #394 for updates :)

**Comment by jeremiah** [comment]
I think that mentioning that SC is an open source project and you can review its license might be good. That is often what I look for when evaluating a project to use.
